packages/e2e-tests/package.json

```diff
@@ -10,7 +10,7 @@
     "test": "playwright test",
     "test:chromium": "playwright test --project chromium --reporter=@replayio/playwright/reporter,line",
     "test:replay-chromium": "playwright test --project replay-chromium --reporter=@replayio/playwright/reporter,line",
-    "test:debug": "DEBUG=1 playwright test",
+    "test:debug": "playwright test --workers 1 --headed",
```

Comment:
No objection to this change but curious what motivates it. Is there any practical difference (other than ... just using the built-in flags?)

Reply:
I could also have renamed `DEBUG` to `TEST_DEBUG` or something. DEBUG in the node ecosystem is already heavily used by https://www.npmjs.com/package/debug which I ran into because I set it to debug a playwright issue and it unexpectedly made the tests run headed which took me a bit to track down.

Reply:
Oh I see. That's interesting.